Bio/PDB/DSSP.py

```diff
@@ -38,29 +38,34 @@
 _dssp_cys = re.compile('[a-z]')
 
 # Maximal ASA of amino acids
-# Values from Sander & Rost, (1994), Proteins, 20:216-226
 # Used for relative accessibility
-MAX_ACC = {}
-MAX_ACC["ALA"] = 106.0
-MAX_ACC["CYS"] = 135.0
-MAX_ACC["ASP"] = 163.0
-MAX_ACC["GLU"] = 194.0
-MAX_ACC["PHE"] = 197.0
-MAX_ACC["GLY"] = 84.0
-MAX_ACC["HIS"] = 184.0
-MAX_ACC["ILE"] = 169.0
-MAX_ACC["LYS"] = 205.0
-MAX_ACC["LEU"] = 164.0
-MAX_ACC["MET"] = 188.0
-MAX_ACC["ASN"] = 157.0
-MAX_ACC["PRO"] = 136.0
-MAX_ACC["GLN"] = 198.0
-MAX_ACC["ARG"] = 248.0
-MAX_ACC["SER"] = 130.0
-MAX_ACC["THR"] = 142.0
-MAX_ACC["VAL"] = 142.0
-MAX_ACC["TRP"] = 227.0
-MAX_ACC["TYR"] = 222.0
+
+residue_max_acc = dict(
+    # Miller max acc: Miller et al. 1987 (http://dx.doi.org/10.1016/0022-2836(87)90038-6)
+    # Wilke: Tien et al., 2013 (http://dx.doi.org/10.1371/journal.pone.0080635)
+    # Sander: Sander & Rost, (1994), Proteins, 20:216-226
+    Miller={
```

Comment:
The rationale can be found in the article that I refer to:
http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080635

This will prevent boundary problems like this:
```
if rel_acc > 1.0:
    rel_acc = 1.0
```